composer.json

```diff
@@ -48,7 +48,7 @@
         "phpmd/phpmd": "^2.7.0",
         "phpunit/phpunit": "^6.5.14",
         "psalm/plugin-phpunit": "^0.5.8",
-        "slevomat/coding-standard": "^4.8.7",
+        "slevomat/coding-standard": "^3.3.0",
```

Comment:
I think we should actually have `"^4.0.0"` (i.e. as it was before #814) - it seems that `slevomat/coding-standard` has not strictly adhered to semantic versioning - [version 4.0](https://packagist.org/packages/slevomat/coding-standard#4.0.0) is in fact compatible with PHP 7.0 whereas [version 4.1](https://packagist.org/packages/slevomat/coding-standard#4.1.0) requires PHP 7.1.  This would also allow a later version (which may have useful improvements) to be installed when running CI scripts against PHP 7.1+.

Reply:
Oh, good research! Updated and repushed.